macros/tests/basic.rs

```diff
@@ -147,3 +147,10 @@ testcases![
     b: 1:3-1:4
 }"
 ];
+
+testcases! [
+    repeat_many_trailing_0 {
+        A = "a"* %% "b";
+    }: A("") => "\
```

Comment:
Please use `"abab"` for testing (if we had multiple testcases per grammar, `"aba"` would also be nice to test).